### PR TITLE
DEVCVL-274 - Added UserView Page

### PR DESCRIPTION
**Pull request by shakil-adv**

### CODE REVIEW

**Comment by kyawthetkt** [comment]
image
@sh Published Collections are still displayed on DRAFTED Section.

**Comment by shakil-adv** [comment]
> image @shakil-adv Published Collections are still displayed on DRAFTED Section.

Can you please provide the userAddress please?

> image @shakil-adv Published Collections are still displayed on DRAFTED Section.

It's because the `isPublished` field for those collection is set to `false` in the database. Please change them accordingly and the UI should respond correctly

**Comment by kyawthetkt** [comment]
> > image @shakil-adv Published Collections are still displayed on DRAFTED Section.
> 
> Can you please provide the userAddress please?
> 
> > image @shakil-adv Published Collections are still displayed on DRAFTED Section.
> 
> It's because the `isPublished` field for those collection is set to `false` in the database. Please change them accordingly and the UI should respond correctly

Thanks. got it and it worked

**Review by kyawthetkt — APPROVED** [review]

File: src/components/header/authDropdown/Auth.js [hunk 1]
```diff
@@ -57,8 +61,21 @@ function Auth() {
         setShowModal(false);
     };
 
+    const handleUserDropdown = ({ key }) => {
```
Comment (jibrail-idris) [inline src/components/header/authDropdown/Auth.js:64]: A `switch` would be a better construct choice.

File: src/components/header/authDropdown/Auth.js [hunk 2]
```diff
@@ -57,8 +61,21 @@ function Auth() {
...
+            navigate(`/user/${user?.get("ethAddress")}/auctions`, {
+                replace: true,
+            });
+        }
+    };
+
     const menu2 = (
-        <Menu>
+        <Menu onClick={handleUserDropdown}>
             <Menu.Item key={1} className="chain-item">
```
Comment (jibrail-idris) [inline src/components/header/authDropdown/Auth.js:79]: The `key` value can be a running counter.

File: src/components/header/authDropdown/Auth.js (hunk 1 above)
Comment (shakil-adv) [reply src/components/header/authDropdown/Auth.js:64]: Yes I agree.

Comment (shakil-adv) [reply src/components/header/authDropdown/Auth.js:64]: Done!

File: src/components/header/authDropdown/Auth.js (hunk 2 above)
Comment (shakil-adv) [reply src/components/header/authDropdown/Auth.js:82]: How can we implement this?

Comment (jibrail-idris) [reply src/components/header/authDropdown/Auth.js:82]: `let i = 0`
for each key
    key={i++}

Comment (shakil-adv) [reply src/components/header/authDropdown/Auth.js]: Got it. done!

**Pull request merged**